dts/bindings/ethernet/nxp,kinetis-ethernet.yaml

```diff
@@ -12,3 +12,6 @@ properties:
       required: true
     interrupts:
       required: true
+    phy-addr:
+      type: int
```

Comment:
Looks like this is missing a `required: true`, no? Either that or you should be using `DT_INST_PROP_OR` in the driver.

Reply:
`required: true` is probably best here. Fixed in latest push.